README.md

```diff
@@ -73,6 +73,19 @@ desirable but not yet implemented.
 Execute `cargo test` to run all tests, or check out the latest
 [CI run](https://cloud.drone.io/erikgrinaker/toydb).
 
+## Debugging
+
+VS Code provides a very intuitive environment for debugging toyDB. Follow these steps to set it up:
+1. Install [CodeLLDB](https://marketplace.visualstudio.com/items?itemName=vadimcn.vscode-lldb) extension
+1. Go to "Run and Debug" tab and initialize launch.json, here is my autogenerated [`launch.json`](https://gist.github.com/leotoff/78e20cb1948e8bb20631271576730865) for reference
+
+The above lets debug `toysql` (from "Run and Debug" tab) and unit tests (using a popup in test files). Debugging the engine is a bit more involved:
+1. Add config arg `"-c=${workspaceFolder}/config/toydb.yaml"` to "Debug executable 'toydb'" configuration in `launch.json`, see [my configuration](https://gist.github.com/leotoff/b940df1b947b7d4e6fd42bd057943fe3) for reference
+1. Create a data directory `sudo mkdir /var/lib/toydb && sudo chown $(whoami) /var/lib/toydb` (the path is specified in `config/toydb.yaml`)
+1. Go to "Run and Debug" tab, select "Debug executable 'toydb'", run
+
+If the last command fails with `Address already in use (os error 48)`, go to Activity Monitor and send the "Quit" signal to `toydb` processes. There might be some lingering processes after running the cluster command.
```

Comment:
This is specific to macOS, consider generalizing this for other OSes -- or at least Linux.

Reply:
Oops, that's a good callout, I'm on it.

Reply:
Resolved, I removed MacOS-specific tools.